backend/benefit/calculator/api/v1/serializers.py

```diff
@@ -27,7 +29,20 @@ class Meta:
 class CalculationSerializer(serializers.ModelSerializer):
     id = serializers.UUIDField(required=False)
     rows = CalculationRowSerializer(
-        many=True, help_text="Calculation rows, generated by the calculator"
+        read_only=True,
+        many=True,
+        help_text="Calculation rows, generated by the calculator",
+    )
+    handler_details = UserSerializer(
+        help_text="The handler object, with fields, currently assigned to this calculation and application (read-only)",
+        read_only=True,
+        source="handler",
+    )
+    handler = serializers.PrimaryKeyRelatedField(
+        required=False,
+        allow_null=True,
+        queryset=User.objects.filter(is_staff=True),
+        help_text="Set the handler of this Calculation (write-only)",
```

Comment:
Can we add `write_only=True` here?

Reply:
Yes, good catch, I think I had it there at some point, wondering how it got lost